source/features/show-user-top-repositories.tsx

```diff
@@ -14,13 +14,9 @@ function buildUrl(queryField: string): URL {
 	return url;
 }
 
-function init(): false | void {
-	const showcaseTitle = select('.js-pinned-items-reorder-container .text-normal');
-	if (!showcaseTitle) {
-		return false;
-	}
-
-	showcaseTitle.firstChild!.after(
+function init(): void {
+	// Showcase title
+	select('.js-pinned-items-reorder-container .text-normal')?.firstChild!.after(
```

Comment:
After all the `?` was needed

Reply:
What do you mean? Make sure you’re using the latest typescript. They changed this behavior in 3.9

Reply:
the `.firstChild!` you cant do after on null
But I solved it another way there in the other PR. (I like that way since with a `?` it will not error if github changes the selector)

Reply:
> (I like that way since with a `?` it will not error if github changes the selector)

You forgot that we **want** RGH to throw errors when GitHub changes the selector...

> But I solved it another way in the other PR.

Undo it

Reply:
> But I solved it another way in the other PR

That's the right way, **not** silencing errors.

Reply:
>  They changed this behavior in 3.9 and now it’s not needed

👆👆👆👆

RGH is still on 3.8, _that_ is the issue.

Reply:
Resolved now